common/src/main/java/de/florianmichael/rclasses/common/access/DataTypeChecker.java

```diff
@@ -38,12 +40,29 @@ public static boolean isInteger(final String input) {
         }
     }
 
+    /**
+     * Checks if the given string is an integer.
+     * @param input The string to check.
+     * @param radix The radix used while parsing the input.
+     * @return True if the string is an integer, false otherwise.
+     */
+    public static boolean isInteger(final String input, final int radix) {
+        if (input.isEmpty() || input.length() > maximumStringSize(radix, Integer.MAX_VALUE)) return false;
+        try {
+            Integer.parseInt(input, radix);
+            return true;
+        } catch (final NumberFormatException e) {
+            return false;
+        }
+    }
+
     /**
      * Checks if the given string is a long.
      * @param input The string to check.
      * @return True if the string is a long, false otherwise.
      */
     public static boolean isLong(final String input) {
+        if (input.isEmpty() || input.length() > 20) return false;
```

Comment:
Can you please add constant fields for all these numbers you added and explain them in java doc?